packages/app-builder/src/components/ui/user/Container.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { styled } from '@linaria/react'
+import { useEditor, useNode } from '@craftjs/core'
+import BREAKPOINT from '../../../utils/breakpoints'
+import { ToolbarItem, ToolbarItemType, ToolbarSection } from '../toolbar'
+
+const ContainerDiv = styled.div`
+  display: flex;
+  align-items: center;
+  justify-items: space-between;
+
+  flex: var(--width);
+
+  @media (max-width: ${BREAKPOINT.Tablet}px) {
+    flex: calc(var(--width) * 2);
+  }
+  @media (max-width: ${BREAKPOINT.MobileL}px) {
+    flex: calc(var(--width) * 3);
+  }
+  @media (max-width: ${BREAKPOINT.MobileS}px) {
+    align-items: flex-start;
+    flex-direction: column;
+    flex: 12;
+  }
+`
+
+const Container = ({
+  padding,
+  children,
+  width,
+  background,
+  height,
+  ...props
+}: {
+  [key: string]: any
+  children?: React.ReactNode
+}) => {
+  const {
+    connectors: { connect, drag },
+    id,
+    // actions: { setProp },
+  } = useNode()
+  const {
+    query: { node },
+  } = useEditor()
+  const isRoot = node(id).isRoot()
+
+  return (
+    <ContainerDiv
+      {...props}
+      ref={(ref) => ref && connect(drag(ref))}
+      isRoot={isRoot}
+      style={{
+        background,
+        height,
+        padding: `${padding}px`,
+        // @ts-ignore
```

Comment:
Does TS not like that? You can just use  the `css` and apply as a className if not...